code/datums/status_effects/debuffs.dm

```diff
@@ -967,3 +967,85 @@
+	var/ants_remaining = 0 //Will act as the main timer as well as changing how much damage the ants do.
+
+/datum/status_effect/ants/on_creation(mob/living/new_owner, amount_left)
+	if(isnum(amount_left) && ants_remaining == 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in ants!</span>")
+		ants_remaining = amount_left
+		RegisterSignal(new_owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	else if(ants_remaining >= 0)
+		to_chat(new_owner, "<span class='userdanger'>You're covered in MORE ants!</span>")
+		if(prob(1)) //Extremely rare
+			new_owner.say("THIS SITUATION HAS BEEN MADE WORSE BY THE ADDITION OF YET MORE ANTS!!", forced = /datum/status_effect/ants)
+		ants_remaining += amount_left
+	. = ..()
+
+/datum/status_effect/ants/on_remove()
+	ants_remaining = 0
+	to_chat(owner, "<span class='notice'>All of the ants are off of your body!</span>")
+	UnregisterSignal(owner, COMSIG_COMPONENT_CLEAN_ACT, .proc/ants_washed)
+	. = ..()
+
+/datum/status_effect/ants/proc/ants_washed()
+	owner.remove_status_effect(STATUS_EFFECT_ANTS)
+	return COMPONENT_CLEANED
+
+/datum/status_effect/ants/tick(delta_time)
+	var/mob/living/carbon/human/M = owner
+	M.adjustBruteLoss(max(0.2, round((ants_remaining * 0.06),0.1))) //More ants means more damage
+	if(!M.stat) //Makes sure people don't scratch at themselves while they're dead
+		if(DT_PROB(15, delta_time))
+			var/ant_scream = pick(1,2)
+			switch(ant_scream)
+				if(1)
+					M.say("GET THEM OFF ME!!", forced = /datum/status_effect/ants)
+				if(2)
+					M.say("OH GOD THE ANTS!!", forced = /datum/status_effect/ants)
+		if(DT_PROB(30,delta_time))
+			M.emote("scream")
+		if(DT_PROB(80, delta_time))
+			switch (rand(1, 50))
+				if (1 to 8) //16% Chance (if I made it 15% the other two probabilities wouldn't be integers)
+					var/obj/item/bodypart/head/hed = M.get_bodypart(BODY_ZONE_HEAD)
+					to_chat(M, "<span class='danger'>You scratch at the ants on your scalp!.</span>")
+					hed.receive_damage(0.1,0)
+				if (8 to 29) //42% chance
+					var/obj/item/bodypart/arm = M.get_bodypart(pick(BODY_ZONE_L_ARM,BODY_ZONE_R_ARM))
+					to_chat(M, "<span class='danger'>You scratch at the ants on your arms!</span>")
+					arm.receive_damage(0.1,0)
+				if (29 to 50) //42% chance
+					var/obj/item/bodypart/leg = M.get_bodypart(pick(BODY_ZONE_L_LEG,BODY_ZONE_R_LEG))
+					to_chat(M, "<span class='danger'>You scratch at the ants on your leg!</span>")
+					leg.receive_damage(0.1,0)
+		if(DT_PROB(3, delta_time)) //See? It's not ALL bad!
+			M.reagents.add_reagent(/datum/reagent/toxin/histamine,rand(1,3))
+			ants_remaining -= 5
+	ants_remaining -= 1
+	if(ants_remaining <= 0)
+		M.remove_status_effect(STATUS_EFFECT_ANTS) //If this person has no more ants on them, they are no longer affected.
+
+/atom/movable/screen/alert/status_effect/ants
+	name = "Ants!"
+	desc = "<span class='warning'>JESUS FUCKING CHRIST! CLICK TO GET THOSE THINGS OFF!</span>"
```

Comment:
```suggestion
	desc = "<span class='warning'>SPACE JESUS FUCKING CHRIST! CLICK TO GET THOSE THINGS OFF!</span>"
```

Reply:
Better yet, make it read your chosen chaplain deity, narsie if you are cultist and heretic gods if you are a heretic